ansible-scylla-node/tasks/ssl.yml

```diff
@@ -24,37 +37,70 @@
         issuer:
           commonName: "{{ scylla_cluster_name }}.internal"
         provider: selfsigned
+      when: ca_crt_details.stat.exists == False
```

Comment:
Current module [docs](https://docs.ansible.com/ansible/latest/collections/community/crypto/x509_certificate_module.html) state that certificate is created if not present, this conditional is unnecessary.

Reply:
IIRC The issue that it creates it when it is already present, but it shouldn't